api/api-template.md

```diff
@@ -0,0 +1,57 @@
+<!-- REMOVE ALL COMMENTS AFTER FILLING OUT TEMPLATE-->
+<!-- Add your function name in the headline below with an accompanying anchor name.  The anchor should be the same as the function name-->
+## function_name() [](function_name) 
+
+<!-- 
+Include a description of the function here.  It should succinctly describe what the function does and any important considerations with regards to function usage.  Include a link to the relevant section in Using TimescaleDB if one exists.
+-->
+
+
+<!-- Include a table with required arguments, including a name and description with type (e.g. REGCLASS, INTEGER)-->
+#### Required Arguments [](function_name-required-arguments)
+
+| Name                 | Description                                       |
+|----------------------|---------------------------------------------------|
+| `sample_argument`    | (TYPE) Quick and clear description of the argument|
+| `another_argument`   | (TYPE) This argument description spans more than one line, but that's ok as long as there is an ending symbol |
+
+
+<!-- Include a table of optional arguments if there are any, otherwise do not include this section in your entry -->
+#### Optional Arguments [](function_name-optional-arguments)
```

Comment:
From readability perspective, thoughts on one argument section that includes required & optional, but label them somehow accordingly?

I find the "separate sections" sometime harder to read

https://docs.timescale.com/latest/api#continuous_aggregate-create_view-with-required

Reply:
I think it'll get messier with a unified section since there would need to be some additional signifier to distinguish between them.  I think that design should play a role in making it easier to parse, controlling spacing between headline and content, maybe choosing a different font size for that level of header, maybe making a `#####` instead of `####` and styling it differently, etc.